### ISSUE TITLE
Support dbengine memory mode

### ISSUE THREAD

**Issue by tullis** [dmcanally/denver-netdata#16]
Hello, thanks for your work on this module.

Is there any chance that you could modify it to support the `memory_mode` of `dbengine`?

https://docs.netdata.cloud/database/engine/

```
Class[Netdata]: parameter 'memory_mode' expects a match for Enum['map', 'none', 'ram', 'save'], got 'dbengine'
```

**Comment by baldurmen** [comment]
This has been fixed by commit 20edc88b969103fa34f42bed1251c4d557c1d72f

**Comment by txdmc** [comment]
Thanks baldurmen!
